### ISSUE TITLE
cudaCheckError() failed : no kernel image is available for execution on the device

### ISSUE THREAD

**Issue by njustymk** [jd730/OICR-pytorch#15]
selective_search
Loaded dataset `voc_2007_trainval` for training
Set proposal method: selective_search
Preparing training data...
voc_2007_trainval ss roidb loaded from /test/data/PASCAL/data/cache/voc_2007_trainval_selective_search_roidb.pkl
done
before filtering, there are 5011 images...
after filtering, there are 5011 images...
voc_2007_trainval ss roidb loaded from /test/data/PASCAL/data/cache/voc_2007_trainval_selective_search_roidb.pkl
5236 71
5011 roidb entries
results/test_single/log/log.txt file already exists
results/test_single/log/log_6.txt
results/test_single/log/progress.csv file already exists
Logging to results/test_single/log
oicr
Loading pretrained weights from data/pretrained_model/vgg16_caffe.pth
Learning rate 0.001
cudaCheckError() failed : no kernel image is available for execution on the device

**Comment by jd730** [comment]
Could you try description on https://github.com/roytseng-tw/Detectron.pytorch/issues/168

e.g. Set
```
CUDA_VISIBLE_DEVICES=0
LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
PATH=/usr/local/cuda/bin:$PATH
```

Also, if you still confront the same error, could you tell me which script you ran, your cuda version, GPU arch version, etc.

**Comment by njustymk** [comment]
> Could you try description on [roytseng-tw/Detectron.pytorch#168](https://github.com/roytseng-tw/Detectron.pytorch/issues/168)
> 
> e.g. Set
> 
> ```
> CUDA_VISIBLE_DEVICES=0
> LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
> PATH=/usr/local/cuda/bin:$PATH
> ```
> 
> Also, if you still confront the same error, could you tell me which script you ran, your cuda version, GPU arch version, etc.

still didn't work
pytorch0.4.0
cuda9.0
cudnn7.0
TITAN V
CUDA_VISIBLE_DEVICES=0 python3 trainval_net.py --dataset pascal_voc --net vgg16 \
   --bs 2 --nw 6 --save_dir='results/test_single'  \
   --lr 0.001 --cuda --disp_interval 50 --vis \
	--checkpoint_interval=500 --model='oicr' --threshold=0.1

**Comment by jd730** [comment]
did you add `-gencode arch=compute_70,code=sm_70`  [here](https://github.com/jd730/OICR-pytorch/blob/6ec17f66e65d971af0b9c5f1655aa6cfbbc99d35/lib/make.sh#L20) before running `make.sh`?

Volta architecture's gencode is 70.

I think the reason why you got an error is this.


cf. when I ran on V100, it worked.

**Comment by njustymk** [comment]
> did you add `-gencode arch=compute_70,code=sm_70` [here](https://github.com/jd730/OICR-pytorch/blob/6ec17f66e65d971af0b9c5f1655aa6cfbbc99d35/lib/make.sh#L20) before running `make.sh`?
> 
> Volta architecture's gencode is 70.
> 
> I think the reason why you got an error is this.
> 
> cf. when I ran on V100, it worked.

It worked by add `-gencode arch=compute_70,code=sm_70`, thank you help me patiently very much!

**Comment by jd730** [comment]
Great!
I add Volta's gencode on ReadMe
